build-system/tasks/visual-diff/helpers.js

```diff
@@ -16,6 +16,7 @@
 'use strict';
 
 const argv = require('minimist')(process.argv.slice(2));
+const puppeteer = require('puppeteer'); // eslint-disable-line no-unused-vars
```

Comment:
Is this imported solely for typechecking? In TS we can do `import type <...>` - is there no equivalent for JSDoc?

Reply:
Yes, solely for typechecking. There's a long discussion on this at https://github.com/Microsoft/TypeScript/issues/14377. Sadly, no resolution yet, and none of the hacks worked.